wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -73,8 +74,8 @@ namespace :wagn do
 
   desc "migrate structure and cards"
   task :migrate =>:environment do
-    ENV['SCHEMA'] ||= "#{Wagn.root}/db/schema.rb"
-    
+    ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
```

Comment:
1. We still use schema.rb in the gem for generating new decks.  Core developers update that schema when there is a new structure migration.  We don't actually need it during structure migrations for decks; the rails pattern is to update the schema.rb file then so that new copies of the app are structurally synched with old ones.
2. We have to write a Gemfile in the decks.  Otherwise the decks can't use bundler to manage their gem updates, including wagn updates.

For now, I'll use the term "rails decks" to decks not connected to a rails app (via decko-rails) and "free decks" for traditional stand-alone ones.

Free decks don't currently have any links to db/migrations in them and I don't see why it would be an improvement to have one. For rails decks, that would be seem admissible if it's the best way to get things working.